tests/readers/test_pytorch.py

```diff
@@ -37,7 +37,7 @@ def test_dataset(
             tmpdir, y_shape, y_sparse, y_key_dim, num_attrs, pass_attrs
         ) as y_kwargs:
             x_array, y_array = x_kwargs["array"], y_kwargs["array"]
-            dataset = PyTorchTileDBDataset(
+            dataset = _PyTorchTileDBDataset(
```

Comment:
Let's remove `test_dataset`, `_PyTorchTileDBDataset` is tested via the `PyTorchTileDBDataLoader`